app/jobs/regular/emit_web_hook_event.rb

```diff
@@ -110,6 +110,15 @@ def web_hook_request(args, web_hook)
           web_hook_event_id: web_hook_event.id,
           event_type: args[:event_type]
         }, user_ids: User.human_users.staff.pluck(:id))
+
+        if SiteSetting.retry_web_hook_events? && response.status != 200
+          args[:retry_count] ||= 0
+          return if args[:retry_count] > 4
```

Comment:
Let's try and avoid in-lining magic numbers. If the number is fixed, we should assign it to a constant which would make it clearer in the future what the number stands for.